terraform/eks/daemon/gpu/main.tf

```diff
@@ -701,19 +701,24 @@ resource "kubernetes_cluster_role_binding" "rolebinding" {
     namespace = "amazon-cloudwatch"
   }
 }
-
 resource "null_resource" "validator" {
   depends_on = [
     aws_eks_node_group.this,
     kubernetes_daemonset.service,
     kubernetes_cluster_role_binding.rolebinding,
     kubernetes_service_account.cwagentservice,
   ]
+
   provisioner "local-exec" {
     command = <<-EOT
-      echo "Validating EKS metrics/logs for EMF"
       cd ../../../..
-      go test ${var.test_dir} -eksClusterName=${aws_eks_cluster.this.name} -computeType=EKS -v -eksDeploymentStrategy=DAEMON -eksGpuType=nvidia
+      i=0
+      while [ $i -lt 10 ]; do
```

Comment:
Why 10? I'm curious how long each iteration will take. should the test itself iterate for x times with some sleep in between?

Reply:
I can reduce it to 5, but the terraform has a timeout of 1 hour already and since each test takes around 3 minutes this will add up to 30 minutes +10/15 minutes for terraform. So its about 5 mintues less than the timeout time. I think 10 is fine